airbyte-integrations/connectors/source-zuora/source_zuora/spec.json

```diff
@@ -15,7 +19,14 @@
       "window_in_days": {
         "type": "integer",
         "description": "The amount of days for each data-chunk begining from start_date. Bigger the value - faster the fetch. (Min=1, as for a Day; Max=364, as for a Year).",
-        "examples": [30, 60, 90, 120, 200, 364],
+        "examples": [
```

Comment:
add minimum and maximum values for this option

Reply:
Sure, done.